src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -18,6 +18,7 @@ public class MarkLogicSourceConfig extends MarkLogicConfig {
 
     public static final String DSL_QUERY = "ml.source.optic.dsl";
     public static final String SERIALIZED_QUERY = "ml.source.optic.serialized";
+    public static final String CONSTRAINT_COLUMN_NAME = "ml.source.optic.constraintColumn.name";
```

Comment:
Just realized there's both "constraintColumn" and "constrainingColumn" - I think the former is a little better, so use that throughout the PR.